src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -12,6 +12,7 @@ import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymou
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
 import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
 import {ANONYMOUS, ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/constant';
+import CommunityChannelItem from '../../../components/ChatList/CommunityChannelItem';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CommunityChannelItem` is a good addition as it's used in the rendering of channels of type 'ANON_TOPIC'. This change improves the functionality and user experience by providing a specific component for anonymous community notifications.
```diff
+ import CommunityChannelItem from '../../../components/ChatList/CommunityChannelItem';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->